LibVLCSharp/Shared/MediaSource.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace LibVLCSharp.Shared
+{
+    public class MediaSource : IMediaSource
+    {
+        static MediaSource()
```

Comment:
Maybe the constructor is worth a warning as comments that it initializes libvlc (and makes native calls). That's the only object from the wrapper that doesn't exist in the C API and makes native calls.